Sources/LiveKit/Core/Room.swift

```diff
@@ -546,3 +549,40 @@ extension Room: EngineDelegate {
         }
     }
 }
+
+// MARK: - AppStateDelegate
+
+extension Room: AppStateDelegate {
+
+    func appDidEnterBackground() {
+
+        guard options.suspendLocalVideoTracksInBackground else { return }
+
+        guard let localParticipant = localParticipant else { return }
+        let promises = localParticipant.videoTracks.values
```

Comment:
if we do these frequently, I wonder if we can make a helper function localParticipant.localVideoTracks that performs these filtering? or if `videoTracks` on localParticipant be changed to return the right types?

Reply:
yup that would make it cleaner 🙂